pkg/errinsrc/srcrender.go

```diff
@@ -20,8 +21,12 @@ const endEscape = "\x1b[0m"
 
 const tabSize = 4
 
-var aurora = auroraPkg.NewAurora(true)
-var enableColors = true
+var aurora auroraPkg.Aurora
+var enableColors bool
+
+func init() {
+	ColoursInErrors(supportscolor.Stdout().SupportsColor)
```

Comment:
This code runs in many different environments including the build system on the web platform, and we don't always pipe things to stdout. This seems like it could easily do the wrong thing as a result.

Probably we'd be better off with a setting to turn it off explicitly, and filtering it out at the time of rendering if we know we're outputting to a terminal that doesn't support colors?

Reply:
True, but this is a step to better behaviour on older windows terminals, other locations explicitly turn the colours on or off as needed (such as the build system)

This init change just moves us from always on to mostly on

Reply:
But that could cause a regression on the platform side, for example

Reply:
What regression are you worried about (i.e. I'm not following)?

The platform doesn't use this package directly, apart from the builder images, which are unix based and their Stdout does support colour. Plus the builder images already have this init function:

```go
func init() {
	errinsrc.ColoursInErrors(true)
	errinsrc.TerminalWidth = 80
}
```